Sources/StreamChatUI/ChatChannelList/ChatChannelListItemView.swift

```diff
@@ -31,54 +31,38 @@ open class _ChatChannelListItemView<ExtraData: ExtraDataTypes>: _View, ThemeProv
     open private(set) lazy var bottomContainer: ContainerStackView = ContainerStackView().withoutAutoresizingMaskConstraints
     
     /// The `UILabel` instance showing the channel name.
-    open private(set) lazy var titleLabel: UILabel = components
-        .channelList
-        .itemSubviews
-        .titleLabel
-        .init()
+    open private(set) lazy var titleLabel: UILabel = UILabel()
         .withoutAutoresizingMaskConstraints
         .withAdjustingFontForContentSizeCategory
         .withBidirectionalLanguagesSupport
     
     /// The `UILabel` instance showing the last message or typing members if any.
-    open private(set) lazy var subtitleLabel: UILabel = components
-        .channelList
-        .itemSubviews
-        .subtitleLabel
-        .init()
+    open private(set) lazy var subtitleLabel: UILabel = UILabel()
         .withoutAutoresizingMaskConstraints
         .withAdjustingFontForContentSizeCategory
         .withBidirectionalLanguagesSupport
     
     /// The `UILabel` instance showing the time of the last sent message.
-    open private(set) lazy var timestampLabel: UILabel = components
-        .channelList
-        .itemSubviews
-        .timestampLabel
-        .init()
+    open private(set) lazy var timestampLabel: UILabel = UILabel()
```

Comment:
All these generic UILabel´s should not come from `Components`. If the customer wants to customise it, simply overrides the `ChatChannelListItemView`.

Reply:
That's true. Already fixed 👍